tests/unit/forklift/test_legacy.py

```diff
@@ -1359,6 +1360,14 @@ def storage_service_store(path, file_path, *, meta):
                 IMetricsService: metrics,
             }.get(svc)
         )
+        db_request.registry.settings = {
+            "warehouse.distribution_table": "example.pypi.distributions"
+        }
+
+        update_database = pretend.stub(
```

Comment:
nit: suggest changing this to "update_bigquery" just to clarify that it isn't our "core" database.